docker-compose.yml

```diff
@@ -1,22 +1,10 @@
 version: '3'
 services:
-  reverseproxy:
-    build: 
-      context: ./services
-      dockerfile: ./reverse-proxy/Dockerfile
-    ports:
-      - "10080:10080"
-    depends_on:
-      - registryservice
-      - gatewayservice
-      - vaultservice
-      - twinservice
-      - historyservice
   registryservice:
```

Comment:
Microsoft.Azure.IIoT.Services.OpcUa.Jobs service is missing

Reply:
Nice catch!!